deploy/servctl_utils/deploy_command_utils.py

```diff
@@ -8,7 +8,7 @@
 import time
 
 from deploy.servctl_utils.other_command_utils import check_kubernetes_context, set_environment
-from hail_elasticsearch_pipelines.kubernetes.kubectl_utils import is_pod_running, get_pod_name, get_node_name, run_in_pod, \
+from deploy.servctl_utils.kubectl_utils import is_pod_running, get_pod_name, get_node_name, run_in_pod, \
```

Comment:
I've been meaning for a while to move the utility functions we need into our own repo instead of importing them from a submodule. I finally bit the bullet here because there is a unicode handling bug that is fixed in the seqr `run` utility but not in the imported one, so I copied over the file we need and changed it to use our working utility function

Reply:
It is good to move the utility file into our repository.